mitxpro/views.py

```diff
@@ -43,10 +43,32 @@ def index(request, **kwargs):  # pylint: disable=unused-argument
     """
     The index view
     """
-    if request.method == "GET":
-        return render(request, "index.html", context=get_js_settings_context(request))
-    else:
-        return redirect(request.get_full_path())
```

Comment:
It would be nice to keep this, though I understand removing this is necessary for the GTM code to work so I guess we have to remove it.